src/lib/sitemap/tutorials-content-urls.ts

```diff
@@ -0,0 +1,48 @@
+import { getAllCollections } from 'lib/learn-client/api/collection'
+import { SectionOption } from 'lib/learn-client/types'
+import { activeProductSlugs } from 'lib/products'
+import { generateTutorialMap } from 'pages/api/tutorials-map'
+import { ProductSlug } from 'types/products'
+import { getCollectionSlug } from 'views/collection-view/helpers'
+import { makeSitemapElement } from './helpers'
+import { Collection as ClientCollection } from 'lib/learn-client/types'
+
+function getTutorialLandingPaths(): string[] {
+	const activeSlugs = activeProductSlugs.map(
+		(productSlug: ProductSlug) => `/${productSlug}/tutorials`
+	)
+	const sectionOptionsWithLandingPage = Object.values(SectionOption).filter(
+		(option: SectionOption) => option === SectionOption.onboarding
+	)
+	const sectionOptions = sectionOptionsWithLandingPage.map(
+		(slug: SectionOption) => `/${slug}`
+	)
+
+	return [...activeSlugs, ...sectionOptions]
+}
+
+async function getCollectionPaths() {
+	const allCollections = await getAllCollections()
+
+	return allCollections.map((collection: ClientCollection) =>
+		getCollectionSlug(collection.slug)
+	)
+}
+
+export async function allTutorialsUrls() {
+	const tutorialLandingSlugs = getTutorialLandingPaths().map((slug: string) =>
+		makeSitemapElement({ slug })
+	)
+
+	const formattedCollectionSlugs = await getCollectionPaths()
+	const collectionSlugs = formattedCollectionSlugs.map((slug: string) =>
+		makeSitemapElement({ slug })
+	)
```

Comment:
⛏️Naming nitpick: I would think the `formatted` version of the slugs would be what comes after the `makeSitemapElement` function. Since the collection paths return from that api client call haven't been formatted at all yet. Thoughts?

Reply:
This makes sense to me. Grab the slugs, format them, output an element with a formatted URL. I'll push a change

Reply:
I'm also going to take it a step further and rename all resulting arrays 'fields' instead of 'urls' to be more inline with next-sitemap API naming: getServerSideSitemap(ctx, **fields**)

For example:
`allDocsUrls` -> `allDocsFields`
`collectionUrls` -> `collectionFields`

Reply:
Sounds great! 👍🏻